metafacture-flux/src/main/java/org/metafacture/flux/HelpPrinter.java

```diff
@@ -171,4 +193,19 @@ private static Collection<String> getAvailableArguments(final Class<?> moduleCla
         return Collections.emptyList();
     }
 
+    @SuppressWarnings("IllegalType")
+    private static HashMap<String, List<String>> loadExamples(final String fname) throws IOException {
+        final File f = new File(fname);
+        if (Files.exists(f.toPath())) {
```

Comment:
Why the indirection?

```suggestion
        if (f.exists()) {
```